test-data/pybind11_mypy_demo/stubgen/pybind11_mypy_demo/basics.pyi

```diff
@@ -11,12 +12,10 @@ class Point:
         radian: ClassVar[Point.AngleUnit] = ...
         def __init__(self, value: int) -> None: ...
         def __eq__(self, other: object) -> bool: ...
-        def __getstate__(self) -> int: ...
         def __hash__(self) -> int: ...
         def __index__(self) -> int: ...
         def __int__(self) -> int: ...
         def __ne__(self, other: object) -> bool: ...
-        def __setstate__(self, state: int) -> None: ...
```

Comment:
the AST-based stub generator specifically excludes `__getstate__/__setstate__`, so I think it makes sense for the inspection-based generator to also exclude these.  I can't think of a circumstance where exposing these within stubs would be useful, but if we decide that it _is_ then I think it makes sense to do it for both types of stub generators (which is kind of the point of this PR).